### ISSUE TITLE
iscsistart not installed

### ISSUE THREAD

**Issue by dualdm** [open-iscsi/open-iscsi#104]
--- /usr/src/open-iscsi/Makefile        2018-03-01 12:13:51.482042302 +0600
+++ Makefile    2018-05-21 13:46:20.000000000 +0600
@@ -18,7 +18,8 @@
 MANPAGES = doc/iscsid.8 doc/iscsiadm.8 doc/iscsi_discovery.8 \
                iscsiuio/docs/iscsiuio.8 doc/iscsi_fw_login.8 doc/iscsi-iname.8 \
                doc/iscsistart.8
-PROGRAMS = usr/iscsid usr/iscsiadm utils/iscsi-iname iscsiuio/src/unix/iscsiuio
+PROGRAMS = usr/iscsid usr/iscsiadm utils/iscsi-iname iscsiuio/src/unix/iscsiuio \
+               usr/iscsistart
 SCRIPTS = utils/iscsi_discovery utils/iscsi_fw_login utils/iscsi_offload \
                  utils/iscsi-gen-initiatorname
 INSTALL = install

**Comment by gonzoleeman** [comment]
Yes, iscsistart used to be less-used. Please submit a pull request.
